UPGRADING.md

```diff
@@ -1,6 +1,20 @@
 Upgrading Hyperclient
 =====================
 
+### Upgrading to >= 0.9.0
+
+Previous versions of Hyperclient performed asynchronous requests using [futuroscope](https://github.com/codegram/futuroscope) by default, which could be disabled by providing the `:async` option to each Hyperclient instance. This has been removed and you can remove any such code.
+
+```ruby
+api = Hyperclient.new('https://grape-with-roar.herokuapp.com/api') do |client|
+  client.options[:async] = false
+end
+```
+
+The default new behavior is synchronous. Include `futuroscope` in your `Gemfile` and wrap Hyperclient requests into `Futuroscope::Future.new` blocks when you wish to utilize this pattern.
```

Comment:
May I suggest we point people at `concurrent-ruby` for the reasons I discussed on #123?
It's equivalent in this case is wrapping the request with `Concurrent::Future.execute`.